flask_app/myswaggerservice.py

```diff
@@ -7,6 +7,11 @@ class MySwaggerService:
     # static SQL conection
     conn = pyodbc.connect('Driver={ODBC Driver 17 for SQL Server};' 'Server=(localdb)\MSSQLLocalDB;' 'Database=account_receivable;' 'Trusted_Connection=yes;')
     
+    # static password salt so that password salt is the same 
+    # for every created password can be retrieved easily for verification
+    salt =  bcrypt.gensalt()
```

Comment:
i think this is not static because it's generated every time the app starts. So every app restart the salt changes and that invalidates the existing passwords. Maybe you should keep a generated salt and set it in a configuration like environment variable.

Reply:
Yes, the salt is not static and I can see the issue...Okay, I will fix it. Thank you for your feedback!

Reply:
If I set the salt as an environment variable, should I also set the database string as an environment variable?

Reply:
yes database connection string should also be configurable so it should be in env var